lib/spack/docs/gpg.rst

```diff
@@ -0,0 +1,43 @@
+.. _gpg:
+
+===========
+GPG support
+===========
```

Comment:
Yeah, it's not super clear. But it's done [here](https://github.com/LLNL/spack/blob/develop/lib/spack/docs/conf.py#L81,L96).